Core/Core/Planner/Model/UseCase/GetPlannables.swift

```diff
@@ -140,15 +177,20 @@ public class GetPlannables: UseCase {
     public func makeRequest(environment: AppEnvironment, completionHandler: @escaping RequestCallback) {
         // If we would send out the request without any context codes the API would return all events so we do an early exit
         if (contextCodes ?? []).isEmpty {
-            completionHandler(.init(plannables: [], calendarEvents: []), nil, nil)
+            completionHandler(.empty, nil, nil)
             return
         }
 
-        if environment.app == .parent {
-            getObserverCalendarEvents(env: environment) { response, urlResponse, error in
-                completionHandler(Response(plannables: nil, calendarEvents: response), urlResponse, error)
+        switch environment.app {
+        case .parent, .teacher:
+
+            getObserverEvents(env: environment) { response, urlResponse, error in
+                let events = response?.compactMap({ $0.event })
+                let notes = response?.compactMap({ $0.note })
+                completionHandler(.make(calendarEvents: events, plannerNotes: notes), urlResponse, error)
             }
-        } else {
+
+        default:
```

Comment:
Could we please use `case .student:` here, for robustness?
In case we add a new app in the future `default` may cause a hidden bug.